ts/sdk/src/competitionClient.ts

```diff
@@ -413,9 +417,43 @@ export class CompetitionsClient {
 
 	public getCompetitionPublicKey(name: string): PublicKey {
 		const encodedName = encodeName(name);
-		return getCompetitionAddressSync(
-			this.program.programId,
-			encodedName
+		return getCompetitionAddressSync(this.program.programId, encodedName);
+	}
+
+	public async getCompetitionDetails(
+		competition: PublicKey,
+		insuranceFundVaultBalance: BN
+	) {
+		const competitionAccount = await this.program.account.competition.fetch(
+			competition
 		);
+
+		const quoteSpotMarketAccount = this.driftClient.getQuoteSpotMarketAccount();
+		const protocolOwnedShares =
+			quoteSpotMarketAccount.insuranceFund.totalShares.sub(
+				quoteSpotMarketAccount.insuranceFund.userShares
+			);
+
+		const protocolOwnedBalance = unstakeSharesToAmount(
+			protocolOwnedShares,
+			quoteSpotMarketAccount.insuranceFund.totalShares,
+			insuranceFundVaultBalance
+		);
+
+		const maxPrize = protocolOwnedBalance
+			.sub(competitionAccount.sponsorInfo.minSponsorAmount)
+			.mul(competitionAccount.sponsorInfo.maxSponsorFraction);
+
+		const prizePools = [
+			BN.min(new BN(1000).mul(QUOTE_PRECISION), maxPrize.div(new BN(10))),
```

Comment:
I think this should be `BN.max()`? or are we capping the max prize of the smallest two pools to be $1000/$5000?

Reply:
capping to those numbers for smaller pools